kornia/__init__.py

```diff
@@ -26,13 +26,12 @@
     adjust_brightness,
 )
 from kornia.contrib import (
-    spatial_soft_argmax2d,
```

Comment:
why are you removing this ?

Reply:
Because I moved moved to kornia/geometry to be together with convsoftargmax, as we discussed before 
https://github.com/ducha-aiki/kornia/blob/rebased-local-feature-detection/kornia/geometry/spatial_soft_argmax.py#L409